serializer/seata-serializer-fury/src/main/java/org/apache/seata/serializer/fury/FurySerializerFactory.java

```diff
@@ -16,42 +16,49 @@
  */
 package org.apache.seata.serializer.fury;
 
-import org.apache.fury.Fury;
-import org.apache.fury.ThreadLocalFury;
-import org.apache.fury.ThreadSafeFury;
-import org.apache.fury.config.CompatibleMode;
-import org.apache.fury.config.Language;
-import org.apache.seata.core.serializer.SerializerSecurityRegistry;
+import org.apache.seata.core.serializer.Serializer;
+
+import java.util.function.Function;
 
 public class FurySerializerFactory {
```

Comment:
I think this class is redundant and I want to **use the following SerializerDelegate to replace it**. Is it possible?